src/Libraries/RevitNodes/Elements/ModelCurve.cs

```diff
@@ -79,11 +79,13 @@ private void InitModelCurve(Autodesk.Revit.DB.Curve crv, bool makeReferenceCurve
             if (mc != null)
             {
                 InternalSetCurveElement(mc);
-                if (!InternalSetSketchPlaneFromCurve(crv))
+                if (InternalSetSketchPlaneFromCurve(crv))
                 {
                     InternalSetCurve(crv);
-                    return;
```

Comment:
I believe we need the return statement here - we only return if can successfully set the plane and potentially the curve.